src/eigsolve/biarnoldi.jl

```diff
@@ -261,8 +261,8 @@ function _bischursolve(f, v₀, w₀, howmany::Int, which::Selector, alg::BiArno
             VQᴴv = -view(Q, :, 1:keep)' * M⁻¹Wᴴv
             WZᴴw = -view(Z, :, 1:keep)' * M⁻ᴴVᴴw
 
-            H[1:keep, 1:keep] += VQᴴv * h[1:keep]' # Ĥ = Sₖₖ + (V * Qₖ)' * ṽℓ₊₁ * h̃ₖ'
-            K[1:keep, 1:keep] += WZᴴw * k[1:keep]' # K̂ = Tₖₖ + (W * Zₖ)' * w̃ℓ₊₁ * k̃ₖ'
+            H[1:keep, 1:keep] += VQᴴv * transpose(h[1:keep]) # Ĥ = Sₖₖ + (V * Qₖ)' * ṽℓ₊₁ * h̃ₖ'
+            K[1:keep, 1:keep] += WZᴴw * transpose(k[1:keep]) # K̂ = Tₖₖ + (W * Zₖ)' * w̃ℓ₊₁ * k̃ₖ'
```

Comment:
Are you sure about those changes? It depends on whether you want `h` and `k` to be the transpose of the row vector in the Krylov factorization "A * V = V * H + r * (row vector)" or the adjoint, but I would prefer the adjoint, which is also how it is in the paper if I recall correctly. I think if you want to change it back to transpose, there are other places where it needs to be changed as well, like where the residuals of the eigenvectors are computed.

Reply:
Ok I'll take a further look later today.

Reply:
I decided to change the Arnoldi implementation to always represent the row vector as the (Hermitian) `adjoint` of a column vector, as this is also how it was written in the documentation of `KrylovFactorization` etc.